src/foam/dao/MDAO.java

```diff
@@ -45,6 +45,10 @@ The base AltIndex has two complete subindexes (each holds the entire DAO).
 public class MDAO
   extends AbstractDAO
 {
+  public MDAO() {
```

Comment:
How is this being used? MDAO always needs an 'of'.

Reply:
Is this so we can clone() it, because we shouldn't be cloneing DAOs.